redbot/cogs/cleanup/cleanup.py

```diff
@@ -541,3 +541,25 @@ def check(m):
             await mass_purge(to_delete, channel)
         else:
             await slow_deletion(to_delete)
+
+    @cleanup.command(name="spam")
+    @commands.guild_only()
+    @commands.bot_has_permissions(manage_messages=True)
+    async def cleanup_spam(self, ctx: commands.Context, number: int = 50):
+        """Deletes duplicate messages in the channel from the last X messages. Default 50."""
```

Comment:
It might not be obvious to the user using this command that it deletes duplicates but leaves one instance of the message, could you perhaps make it a bit more clear?